packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.tsx

```
@@ -624,17 +601,23 @@ class NodeBalancerConfigPanel extends React.Component<CombinedProps> {
       disabled
     } = this.props;
 
+    // We don't want to end up with nodes[3].ip_address as errorMap.none
+    const filteredErrors = errors
+      ? errors.filter(
+          thisError =>
+            !thisError.field || !thisError.field.match(/nodes\[[0-9+]\]/)
+        )
+      : [];
+
     const errorMap = getErrorMap(
       [
         'algorithm',
         'check_attempts',
         'check_body',
         'check_interval',
-        'check_passive',
```

This and cipher_suite weren't being read anywhere, so if an error came back with that field it would be swallowed.